exchanges/margin/margin_types.go

```diff
@@ -74,20 +74,20 @@ const (
 	// Unset is the default value
 	Unset = Type(0)
 	// Isolated means a margin trade is isolated from other margin trades
-	Isolated Type = 1 << (iota - 1)
+	Isolated Type = 1 << iota
```

Comment:
Better to revert this, as this skips the first bit `00000010` and you only have 7 entries if you want to add more.

Reply:
Nit:
Would read slightly saner if we move comments to line ends and use the same type declaration.
Then values and comments would line up nicely.
```
const (
	Unset    Type = 0
	Isolated          = 1 << (iota - 1) // Margin is isolated from other trades
	Multi                                          // Margin is shared amongst trades and assets; Cross-Margin
        NoMargin                                  // No Margin; Trade is unleveraged
        SpotIsolated                             // Margin shared amongst all spot trades but isolated from other assets
```